cmd/wallet-web/src/router/routes.js

```diff
@@ -77,27 +77,22 @@ export default [
         ],
       },
       {
-        path: 'oidc',
-        name: 'oidc',
+        path: 'oidc/initiate',
+        name: 'oidcInitiate',
         component: load('layouts/OIDCLayout'),
         meta: { requiresAuth: true, signin: true, disableCHAPI: true, isNavbarHidden: true },
-        children: [
-          {
-            path: 'share',
-            name: 'share',
-            component: load('layouts/OIDCShareLayout'),
-          },
-          {
-            path: 'save',
-            name: 'save',
-            component: load('layouts/OIDCSaveLayout'),
-          },
-          {
-            path: 'initiate',
-            name: 'initiate',
-            component: load('layouts/OIDCInitiateLayout'),
-          },
-        ],
+      },
+      {
+        path: 'oidc/share',
+        name: 'oidcShare',
+        component: load('layouts/OIDCShareLayout'),
+        meta: { requiresAuth: true, signin: true, disableCHAPI: true, isNavbarHidden: true },
+      },
+      {
+        path: 'oidc/save',
+        name: 'oidcSave',
+        component: load('layouts/OIDCSaveLayout'),
+        meta: { requiresAuth: true, signin: true, disableCHAPI: true, isNavbarHidden: true },
```

Comment:
This does not look right. Since `oidc/` is the base for all three of them, it would make more sense to me to have one parent `oidc` route with 3 children. Similar to how `DashboardLayout` has a number of children

Reply:
updated

Reply:
@birtony I remember last time you suggested to have them separated since they behave completely different when accessed, and there is no default view for `/oidc`. Can we discuss this?